internetarchive/openlibrary
# Instrument registration flow

In support of #8863


1. Update our `Sentry` objects to allow for capturing handled exceptions
2. Creates new `OLAuthenticationError`, which are now thrown on IA account creation failures
3. Publishes `/account/create` `POST` handler to Sentry

### Technical


### Testing

1. Deploy this to `testing`
2. Trigger an error during account registration
3. Check Sentry for an `OLAuthenticationError`

### Screenshot


### Stakeholders
